NonSucking.Framework.Serialization/Templates/Attribute/NoosonIncludeAttribute.cs

```diff
@@ -3,7 +3,7 @@
 namespace NonSucking.Framework.Serialization
 {
     [AttributeUsage(AttributeTargets.Field, Inherited = false, AllowMultiple = false)]
-    internal class NoosonIncludeAttribute : Attribute
+    public class NoosonIncludeAttribute : Attribute
```

Comment:
Why did you make this public?
```suggestion
    internal class NoosonIncludeAttribute : Attribute
```